build.sbt

```diff
@@ -14,18 +14,23 @@ scalacOptions ++= Seq("-deprecation", "-feature")
 libraryDependencies ++= Seq(
   "org.apache.spark" %% "spark-core" % sparkVersion % "provided",
   "org.apache.spark" %% "spark-sql" % sparkVersion % "provided",
-  "com.github.music-of-the-ainur" %% "almaren-framework" % "0.2.3-2-4" % "provided",
+  "com.github.music-of-the-ainur" %% "almaren-framework" % "0.9.0-2.4" % "provided",
   "com.typesafe.scala-logging" %% "scala-logging" % "3.9.0",
   "org.scalatest" %% "scalatest" % "3.0.5" % "test",
-  "neo4j-contrib" %% "neo4j-connector-apache-spark" % "4.0.0"
+  "com.modak.neo4j" %% "neo4j-connector-apache-spark" % "4.0.2_for_spark_2.4"
 )
 
 enablePlugins(GitVersioning)
 
-resolvers += ("Spark Packages Repo" at "http://dl.bintray.com/spark-packages/maven").withAllowInsecureProtocol(true)
 resolvers +=
   "Sonatype OSS Snapshots" at "https://oss.sonatype.org/service/local/repositories/releases/content"
 
+ThisBuild / useCoursier := false
+
+resolvers += "nabu-releases" at "https://pkgs.dev.azure.com/nabu-backend/Nabu-Bots/_packaging/nabu-releases/maven/v1"
```

Comment:
we don't need this.

Reply:
We need them because the packages for neo4j spark connector in the maven are not working . We have taken the shaded jars and published to azure artifacts and used in the neo4j.almaren code.

for this package , we need them 

“com.modak.neo4j" %% "neo4j-connector-apache-spark" % "4.0.2_for_spark_2.4"